python/lsst/jointcal/jointcal.py

```diff
@@ -264,6 +264,11 @@ class JointcalConfig(pexConfig.Config):
         dtype=float,
         default=5.0,
     )
+    failOnChi2Increase = pexConfig.Field(
+        doc="Ratio of old/new chi2 to fail the fit on (large chi2 increases are generally non-recoverable.",
+        dtype=float,
+        default=10.0
```

Comment:
The JIRA issue suggests `1e5` here. How did you land on `10`? Answering with a comment in the code would be best.